### PR TITLE
Add Logo

### PR DESCRIPTION
**Pull request by s9105947**
This PR adds the logo as built by @tilsche.
I have added some finishing touches (split across files, add notes, make transparent).

Find additional notes in `share/logo`.

The logos require inkscape to build (and edit), which I'd consider quite a heavy dependency, so the rendered PNGs are uploaded into the source repo as well.

### CODE REVIEW

**Comment by s9105947** [comment]
lo2s logo full version

**Review by bmario — COMMENTED** [review]
Is it only me (or firefox), but in `circle_full.svg` and `full.svg`, the circle is missing.

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 # `lo2s` - Lightweight Node-Level Performance Monitoring
 
+<img src="share/logo/full.png" alt="lo2s logo" width="300"/>
```
[the remark this answers is not in the document]
Comment (s9105947) [reply README.md:7]: > Is it only me (or firefox), but in `circle_full.svg` and `full.svg`, the circle is missing.

The svgs are only to be compatible with inkscape. (b/c they are inkscape svgs)
This is correct, as the circle is in `circle.svg` and github moves the URLs in such a way that the references get broken.

Web should use the png versions for compatibility.

[the remark this answers is not in the document]
Comment (s9105947) [reply README.md]: [7663a44](https://github.com/tud-zih-energy/lo2s/pull/250/commits/7663a4447e8e2c3dfa2ffefe91f2a90030471ef2)

**Review by bmario — APPROVED** [review]
LGTM

**Pull request merged**